tf-modules/cumulus_distribution/iam.tf

```diff
@@ -53,6 +53,13 @@ data "aws_iam_policy_document" "lambda_distribution_api_gateway_policy" {
     resources = [for b in local.allowed_buckets: "arn:aws:s3:::${b}/*"]
   }
 
+  statement {
+    actions   = [
+      "s3:PutObject"
+    ]
+    resources = ["arn:aws:s3:::${var.system_bucket}/*"]
```

Comment:
Is this overly permissive?  I'm wondering if we should explicitly set a keypath to not give access to anything this api doesn't need.